doc: Link to limitations documentation for Softdevice Controller (#5970)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

SoftDevice Controller
=====================

> When working with the SoftDevice Controller, you should be aware of the following known issues.
> In addition, see :ref:`softdevice_controller_limitations` for permanent limitations.

Reviewer: I think this would read better as one sentence:
```suggestion
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
```
